server.js

```diff
@@ -1,24 +1,25 @@
 const express = require('express');
 const exphbs = require('express-handlebars');
+router = express.Router();
 
 const app = express();
 const PORT = process.env.PORT || 8080;
-const home = require('./router/home'); // rename this to router
-const about = require('./router/about'); // rename this to router
-const contact = require('./router/contact'); // rename this to router
+const home = require('./router/home/index.js'); // rename this to router
```

Comment:
So the convention is to usually name the folder routes. It would have been easier to change these three lines rather that changing the dir.

Reply:
Also, the index.js at the end of these three lines are not necessary.